hack/tools/logcheck/main.go

```diff
@@ -35,6 +38,7 @@ var Analyzer = &analysis.Analyzer{
 }
 
 func main() {
+	handleErrors()
 	singlechecker.Main(Analyzer)
```

Comment:
I think the error is coming from `singlechecker.Main(Analyzer)` and it is exiting with exit code 1 while package loading. `singlechecker.Main(Analyzer)`  would be called anyways after handleErrors(), so it will fail?

Reply:
Yeah but it's either that or we error out from the checker.